src/gistUploader.js

```
@@ -1,11 +1,42 @@
 const fs = require('fs');
+const axios = require('axios');
+const GIST_URL = 'https://api.github.com/gists';
 
+function uploadGist(input, description='Gist description') {
+  axios.post(
+    GIST_URL,
+    {
+      'description': description,
+      'public': true,
+      'files': {
+        'Gist':  {
+          'content': input,
+        }
+      }
+    })
+    .then((response) => {
+      console.log(response.data.url);
```

maybe instead of console logging here, you should return a string that you display in main, containing pass/fail/url or something informative...